spec/unit/puppet/provider/registry_value_spec.rb

```diff
@@ -51,7 +51,12 @@
 
   describe "#exists?" do
     it "should return true for a well known hive" do
-      reg_value = type.new(:path => 'hklm\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareType', :provider => described_class.name)
+      reg_value = type.new(:title => 'hklm\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareType', :provider => described_class.name)
```

Comment:
All the other tests in this file use `:path` rather than `:title` (given `:path` is the `namevar`) ... I assume this ties into the next PR? Just a little peculiar to see a different attribute for the type without any explanation for why `:title` is used here rather than `:path`.

Reply:
copy-pasta typo.  Should've been path